en/Editing and formatting/Folding.md

```diff
@@ -0,0 +1,16 @@
+Learn how to get a better overview of large notes by using _folding_ to hide parts of the note. Folding is useful when creating outlines for your notes and when you want to focus on what you're working on at the moment.
+
+You can fold headings and indented lists by hovering the mouse cursor over the section you want to fold, and then selecting the arrow on the left. Folded sections show an arrow regardless of if you hover it or not.
+
+Folding is turned on by default. To turn off folding, open **Settings** → **Editor**, and then turn off **Fold indent** or **Fold heading**, depending on your needs.
+
+To toggle all sections at the same time, use the following commands:
+
+- To collapse all sections, open the [[Command palette]], and then select **Fold all headings and lists**.
+- To expand all sections, open the [[Command palette]], and then select **Unfold all headings and lists**.
+
+> [!tip]
+> If you prefer to fold using your keyboard, you can assign [[Use hotkeys|hotkeys]] to the **Fold less** and **Fold more** commands.
+>
+> - **Fold less** unfolds the section at the text cursor.
+> - **Fold more** folds the previous section closest to the text cursor.
```

Comment:
```suggestion
> - **Fold more** folds the previous section closest to the text cursor.
>
> Successive uses of these commands will repeat the command in the parent context.
```

Phrasing here I'm not sure of, but it should be clear that the commands can be repeated and the scope of the fold grows with each command.

Reply:
```suggestion
> - **Fold more** folds the previous section closest to the text cursor.
>
> You can repeat either command multiple times to fold or unfold the next section.
```

How about this? If possible, I'd like to avoid using "parent context" in favor of a more plain alternative.

Reply:
Hmm, I think this makes it more confusing sadly. Probably because we're combining "fold" and "unfold" in the same sentence so the behavior starts to sound ambiguous.

What I'm hoping to articulate:

- Fold more. Successive uses of this command will scan upward and find the closest unfolded region and fold it.
- Fold less. Successive uses of this command will find the closest folded region of text and unfold it.

"Closest" is still a bit too ambiguous for my liking but that's kinda what I would like to articulate.

Reply:
As I play with it more, "Fold less" seems a bit finicky. It only unfolds if the text cursor is at the end of the line with the folded section, otherwise Fold less does nothing. 

I'm thinking we may not need to explain in detail how it works here, and instead focus on the value of repeating the commands. For example:

> You can repeat the **Fold more** command to incrementally hide more of your note.

Reply:
> You can repeat the Fold more command to incrementally hide more of your note

That's a good way to put it!

> It only unfolds if the text cursor is at the end of the line with the folded section, otherwise Fold less does nothing.

Oh good catch. That's a bug! I really only use **fold less** right after doing fold more so I never noticed.